### ISSUE TITLE
artwork for sim screen buttons

### ISSUE THREAD

**Issue by emily-phet** [phetsims/ratio-and-proportion#84]
Let's put ideas here - based on complexity will ask if Amanda/Amy/Cheryl can take on creating the images. 

**Please take 10 minutes or less before Tuesday 11am MT, and add comment with any suggestions or thoughts.**

- [x] @zepumph 
- [x] @BLFiedler 
- [ ] @Matthew-Moore240 
- [x] @terracoda 
- [x] @jbphet

My thoughts so far:

Explore Screen:
- Two hands side by side
- One hand only

Create Screen:
- Two number spinners
- One number spinner

**Comment by BLFiedler** [comment]
I like the idea of keeping it to the imagery we already use in the simulation, so as simple as two hands works for me. Side by side or slightly offset? Green background or white background? I like two hands more than one since the key is the relationship between the two hands.

Since you can always see the Screen 2 artwork, we'd perhaps want to be careful what the displayed symbols are on the two number spinners were. I like that idea if they were maybe question marks? Though that might scream "MYSTERY GAME!" or something. Maybe one number spinner is better and then the presence of a number doesn't really give much away.

**Comment by zepumph** [comment]
Screen 1:
Two hands, slightly offset with some color background (not white)

Screen 2:
* Exact same thing as screen 1 but with grid lines/units (I worry about that cueing the issues that @BLFiedler just stated above)
* A hand next to a number spinner. Perhaps a bit confusing
* A hand next to a fraction (not number spinners), sketched out like this:
![image](https://user-images.githubusercontent.com/6856943/86629469-f32c4680-bf77-11ea-9d7e-0890030e8649.png)

* With a different background color needed, what about just a screenshot of the "My Challenge" control:
![image](https://user-images.githubusercontent.com/6856943/86629602-24a51200-bf78-11ea-836c-2c59bdfb63ff.png)

**Comment by zepumph** [comment]
I also really like looking at FAMB for context, I like that it uses exclusively views from the sim, perhaps that sim is just lucky that it works out that way so easily:
![image](https://user-images.githubusercontent.com/6856943/86631251-5a4afa80-bf7a-11ea-9b43-4300ee18383a.png)

**Comment by terracoda** [comment]
I am on board with these ideas, too.

I think @zepumph ideas will work really well with the words we are currently using:

**Explore**
![explore-woth-yellow-arrows](https://user-images.githubusercontent.com/1902116/86632795-69e23780-bfaa-11ea-9c22-451f4ce84d5a.png)

**Explore**
![explore-no-arrows](https://user-images.githubusercontent.com/1902116/86632802-6babfb00-bfaa-11ea-88aa-3428ce8c4957.png)

**Create**
![create-with-hover-blue](https://user-images.githubusercontent.com/1902116/86633090-c9d8de00-bfaa-11ea-93fb-93f9c39a9678.png)

**Create**
![create-changed](https://user-images.githubusercontent.com/1902116/86633313-16241e00-bfab-11ea-9d50-564ad7f1717b.png)

I think the Create image should have the hover style on one of the number spinners, and maybe different numbers than the starting numbers

**Comment by jbphet** [comment]
I think I've missed the deadline, but if not, here's a thought for capturing the proportion idea with imagery that is mostly available in the sim and that would be reasonably easy to generate programmatically:

![image](https://user-images.githubusercontent.com/3082831/86840202-b2a7f800-c05f-11ea-9b36-6e381dd83286.png)

The "Create" screen ideas above seem fine to me.

**Comment by emily-phet** [comment]
Here's what I'm thinking, after digesting and adding to the ideas listed above.

**Explore Screen:**
- Two hands showing, one obviously higher than the other, but not at the same scale as the sim (trying to show them as large as possible with still clear difference in height) 

(I had this other thought, but think it's not worth the effort, since it will mostly be viewed at a very small size in nav bar - but graphic is done so that it appears the two hand are moving upwards? So, a static image, but there is an effect with cascading hands that makes it look like the hands are being depicted in motion. Not sure what that effect is called...but it's a common one.)

**Create Screen:**
- Just like @terracoda last set of suggestions here https://github.com/phetsims/ratio-and-proportion/issues/84#issuecomment-654427481, with the ratio at 1:1 and the right hand hover state showing blue. If that somehow looks out of scale with the two hands, we could try one number spinner, showing hover state as @terracoda suggested.

@amanda-phet Suggestions on who might be the right person (expertise and time available) to make a first draft of these icons?

**Comment by amanda-phet** [comment]
I wanted to take a stab at making the hands look as though they are moving, but it's tricky. If you want this to look better, you should probably work with Cheryl. Otherwise, these are incredibly simple and I'm happy to take on the icons. 

Attempt 1: 
![image](https://user-images.githubusercontent.com/7852916/86959647-71751e00-c11b-11ea-9674-17bc22310305.png)


Attempt 2: 
![image](https://user-images.githubusercontent.com/7852916/86959670-7afe8600-c11b-11ea-9e20-bcef00a6c85c.png)

(I personally like one hand having movement with the other looking stationary.)

**Comment by amanda-phet** [comment]
I often like to make different icons for the navbar since they are so much smaller. So here is my first stab. If you'd like to refine the "moving" hand, I would recommend working with Cheryl. If you just want to make minor tweaks to what I have here, please let me know and I'm happy to make the changes. The second icon could possibly be made in code by @zepumph instead of using a screenshot. 

![image](https://user-images.githubusercontent.com/7852916/86960829-573c3f80-c11d-11ea-929d-76b7a354ff7d.png)

[Ai file](https://github.com/phetsims/ratio-and-proportion/files/4892772/RaP-screen-icons.ai.zip)

**Comment by emily-phet** [comment]
Thanks @amanda-phet!

I'd like to include this in the next Thursday design meeting we request, coming in a couple of weeks.

**Comment by terracoda** [comment]
I like attempt 1 :-)

**Comment by emily-phet** [comment]
@zepumph - Can you implement the following?
I'm thinking like this mockup with a few changes I couldn't draw in: [Presentation1.pdf](https://github.com/phetsims/ratio-and-proportion/files/5156440/Presentation1.pdf)

- [ ] For the first screen, like the mockup,  but with the hands the yellow challenge color, with a medium black stroke around outside of hands:
- [ ] For the second screen, like the mockup but with the hands the blue color of the number spinners, and the "number 1" spinner has both the up and down arrows blue.

**Comment by zepumph** [comment]
I added the first screen icon above. It isn't yellow currently. That feature will be on hold until #57 is sorted out. It uses the same icon creation code as in the "My Challenge" accordion box on the create screen. It seemed easiest to handle the color updates for the whole sim at the same time.

**Comment by zepumph** [comment]
![Uploading image.png…]()

**Comment by zepumph** [comment]
@emily-phet, it ended up being challenging to include the hover behavior on the increment button on the second NumberPicker. If we need that feature, we can work on adding that ability to NumberPicker.

Please review.

**Comment by zepumph** [comment]
![Uploading image.png…]()

**Comment by zepumph** [comment]
From design meeting today, it would be good to try to include this hover behavior in the icon. I'll take a look into doing that. 

Also we like having different numbers, like maybe 3/2:

![image](https://user-images.githubusercontent.com/6856943/94596307-6fad8900-0238-11eb-9701-f80dcc2ef0a7.png)


@emily-phet mentioned that maybe this different set of numbers will hint to come to this screen second.

**Comment by zepumph** [comment]
@emily-phet, please review:

![image](https://user-images.githubusercontent.com/6856943/95267632-2cc45600-07e2-11eb-89c9-6a2e9b5da11d.png)

**Comment by emily-phet** [comment]
@zepumph I think the second screen has too much in it...the images look too small when in the nav bar, etc.

Let's go with just using the number spinners alone for the second screen.

**Comment by terracoda** [comment]
I think simplifying to number spinners is a good idea.

**Comment by zepumph** [comment]
Like that?

![image](https://user-images.githubusercontent.com/6856943/95777566-e86a0780-0c72-11eb-8cda-1a6167084745.png)

**Comment by terracoda** [comment]
I like the color added to the hands. That will be a color of the challenge.

**Comment by emily-phet** [comment]
I'm not loving the green/red and green/blue combinations. Maybe a white background would be better?

**Comment by zepumph** [comment]
Like this? 

![image](https://user-images.githubusercontent.com/6856943/95891259-5117b980-0d31-11eb-97ab-c223c4b74938.png)

**Comment by zepumph** [comment]
Oh you meant for both. 

![image](https://user-images.githubusercontent.com/6856943/95891394-7e646780-0d31-11eb-9486-efa931ddb9ee.png)

**Comment by emily-phet** [comment]
This looks great! Closing - the only remaining changes would be having the second screen button match the color of things on the second screen if we change them.
